RecoPixelVertexing/PixelTrackFitting/interface/RiemannFit.h

```diff
@@ -1,17 +1,13 @@
 #ifndef RECOPIXELVERTEXING_PIXELTRACKFITTING_RIEMANNFIT_H
 #define RECOPIXELVERTEXING_PIXELTRACKFITTING_RIEMANNFIT_H
 
```

Comment:
Should this include guard be renamed as well? (Doesn't really matter to me, but I saw other similar guards renamed)